package.json

```
@@ -1,12 +1,15 @@
 {
   "name": "palavreco",
-  "version": "1.0.0",
-  "description": "Um bot de Discord que simula o Wordle",
-  "main": "bot.js",
+  "version": "2.0",
```

[Semver](https://semver.org) requires 3 components.
```suggestion
  "version": "2.0.0",
```